lib/rouge/lexers/sql.rb

```diff
@@ -115,7 +115,7 @@ def self.keywords_type
         rule %r/"/, Name::Variable, :double_string
         rule %r/`/, Name::Variable, :backtick
 
-        rule %r/\w[\w\d]*/ do |m|
+        rule %r/[[:alpha:]][\w]*/ do |m|
```

Comment:
We lose the digit piece here. Did you decide that's unnecessary since `keyword_types` don't include any digits? Similarly, `:alpha:` has slightly different meaning that `\w` - why the change? I actually wonder why we need both pieces anyway. Isn't `%r/\w+/` equivalent? I'm slightly hesitant, though, as maybe there's something I'm not considering. Appreciate your insight.